3-enrich/stream-enrich/examples/config.hocon.sample

```diff
@@ -41,6 +43,27 @@ enrich {
     secret-key: "iam"
   }
 
+  # config for NSQ sink and NSQ source
+NSQ {
+  #Topic name for good event source
+  good-source-topic: "{{sinkKinesisNsqSourceGoodTopicName}}"
+
+  #Channel name for good event source
+  good-source-channel: "{{sinkKinesisNsqSourceGoodChannelName}}"
+
+  #Topic name for bad event sink
+  bad-sink: "{{sinkKinesisNsqSinkTopicBadName}}"
```

Comment:
for the 3 properties above, why aren't you using the configuration in `streams`, i.e.:

- `in.raw`
- `out.enriched`
- `out.bad`

?